runhouse/main.py

```diff
@@ -903,6 +964,27 @@ def stop(
         subprocess.run(RAY_KILL_CMD, shell=True)
 
 
+@server_app.command("status")
+def server_status(
+    cluster_name: Optional[str] = typer.Argument(
+        None, help="A *saved* remote cluster object to restart."
+    ),
+):
+    """Check the HTTP server status on the cluster."""
+    logger.debug("Checking the server status.")
+
+    current_cluster = get_cluster_or_local(cluster_name=cluster_name)
```

Comment:
this command won't work if running `runhouse server status` on a cluster, which i think we should also support

Reply:
looking good to me, would you mind to verify it on your laptop locally? 
![image](https://github.com/user-attachments/assets/60deed53-7ff9-4893-b44a-5a28027d3605)

Reply:
yep good for me now too